docs/testing/testing.md

```diff
@@ -32,10 +32,92 @@ $ yarn test:integration
 
 You can also specify which tests to run by giving a path param, like so: `yarn test:unit packages/<app-name>/tests/unit/path/to/test.spec.js`.
 
-### Acceptance Tests
+### E2E Tests (Playwright)
+
+Our end-to-end test suite is built upon [Playwright Framework](https://github.com/microsoft/playwright), 
+which makes it easy to write tests, debug them and have them run cross-browser with minimal overhead.
+
+#### Prerequisites
+
+To run acceptance tests with Docker, please make sure you have the following tools installed:
+
+- docker
+- docker-compose
+- yarn
+- node
+
+Please also make sure to point `http://host.docker.internal/` to `127.0.0.1` by adding it to your hosts.
+
+#### Prepare & start web
+
+Bundle the web frontend, which then gets mounted into the respective backends. It also gets recompiled on changes.
+
+```shell
+$ yarn build:w
+```
+
+#### Start Docker
+
+Using compose, you can start the required Docker containers by running
+
+For running the test with oc10 run
+```shell
+$ docker compose up oc10
+```
+
+For running the test with ocis run
+```shell
+$ docker compose up ocis
+```
+
+and make sure there are no conflicting ports and everything runs smoothly. You can check if everything has worked by opening [https://host.docker.internal:9200](https://host.docker.internal:9200) (oCIS) and [http://host.docker.internal:8080](http://host.docker.internal:8080) (OC10) and logging in using the demo user (admin/admin).
+
+#### Run E2E tests
+
+Depending on the backend you want to run the tests on, you can either run
+
+```shell
+$ yarn test:e2e:cucumber tests/e2e/cucumber
+```
+
+for an ownCloud 10 backend or
+
+```shell
+$ OCIS=true yarn test:e2e:cucumber tests/e2e/cucumber
+```
+
+for an oCIS backend.
+
+#### Options
+
+To run a particular test, simply add the feature file and line number to the test command, e.g. `yarn test:e2e:cucumber tests/e2e/cucumber/shareFileJourney.feature:13`
+
+Various options are available via ENV variables, e.g.
+- `OCIS=true` to run the E2E tests against an oCIS backend
+- `RETRY=n` to retry failures `n` times
+- `SLOW_MO=n` to slow the execution time by `n` milliseconds
+- `TIMEOUT=n` to set tests to timeout after `n` milliseconds
+- `HEADLESS=bool` to open the browser while the tests run (defaults to true => headless mode)
+- `BROWSER=name` to run tests against a specific browser. Defaults to Chrome, available are Chrome, Firefox, Webkit, Chromium
+
+For debugging reasons, you may want to record a video or traces of your test run. 
+Again, you can use the following ENV variables in your command:
+
+- `REPORT_DIR=another/path` to set a directory for your recorded files
+- `REPORT_VIDEO=true` to record a video of the test run
+- `REPORT_HAR=true` to save request information from the test run
+- `REPORT_TRACING=true` to record traces from the test run
+
+To then open e.g. the tracing, run
+
+```shell
+$ npx playwright show-trace path/to/file.zip
```

Comment:
maybe we have to mention somehow that the files live inside the reports dir... e.g. `npx playwright show-trace $REPORT_DIR/to/file.zip`